libs/platform/user/ebpf_leak_detector.cpp

```diff
@@ -36,7 +36,6 @@ void
 _ebpf_leak_detector::dump_leaks()
 {
     std::unique_lock<std::mutex> lock(_mutex);
-    ebpf_assert(_allocations.empty());
```

Comment:
Move assert until after it dumps the leaked stacks. Assert should still fail the test if we leak memory,

Reply:
fixed